app/views/PDFView.scala.html

```diff
@@ -0,0 +1,122 @@
+                <h2 class="govuk-heading-m">@messages("pdf.logoTitle")</h2>
+            </div>
+            <div class="govuk-grid-column-two-thirds">
+                <h3 class="govuk-heading-m print-header__heading">
+                    @messages("pdf.previewTitle")
+                </h3>
+            </div>
+        </div>
+        <div class="grid-row">
+            <h3 class="govuk-heading-s govuk-!-static-margin-bottom-1">@messages("pdf.companyName")</h3>
+            <p class="govuk-body">SAMPLE LTD</p>
+        </div>
+        <div class="grid-row">
+            <h3 class="govuk-heading-s govuk-!-static-margin-bottom-1">@messages("pdf.utr")</h3>
+            <p class="govuk-body">123456</p>
+        </div>
+        <div class="grid-row print-banner">
+            @displayBanner(calculatorResult)._2
+        </div>
+        <div class="grid-row">
+            <div class="govuk-grid-column-full">
+                <div class="grid-row">
+                    <div class="govuk-grid-column-two-thirds govuk-!-padding-left-0">
+                        <div class="grid-row">
+                            <div class="govuk-grid-column-full">
+                                @displayYourDetails(calculatorResult, accountingPeriodForm, taxableProfit, distributions, associatedCompanies, true)
+                            </div>
+                        </div>
+                        <div class="grid-row">
+                            <div class="govuk-grid-column-full">
+                                <h3 class="govuk-heading-s">Your accounting period covers 2 HMRC financial years</h3>
+                                <p class="govuk-body">2022 to 2023: 1 January 2024 to 31 March 2024<br>
+                                    2023 to 2024: 1 April 2024 to 31 December 2024</p>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="govuk-grid-column-one-third govuk-!-padding-right-0">
+                        <div class="got-ni-box">
+                            <h3 class="govuk-heading-s got-ni-box-border">@messages("pdf.aboutThisResult")</h3>
+                            <h4 class="govuk-heading-xs">@messages("pdf.dataOfResult")</h4>
+                            <p class="govuk-body got-ni-box-border">18 September 2022 10:00:00 UTC</p>
```

Comment:
This timestamp should be the current date time

Reply:
Added timestamp